src/test/java/com/wikia/webdriver/common/core/imageutilities/ImageEditor.java

```diff
@@ -1,19 +1,22 @@
 package com.wikia.webdriver.common.core.imageutilities;
 
+import com.wikia.webdriver.common.core.configuration.Configuration;
 import org.apache.commons.io.FileUtils;
 import org.openqa.selenium.Point;
 import org.openqa.selenium.WebDriverException;
 
+import javax.imageio.ImageIO;
 import java.awt.image.BufferedImage;
 import java.io.File;
 import java.io.IOException;
+import java.util.Map;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import javax.imageio.ImageIO;
-
 public class ImageEditor {
 
+  private static final int dpr = getDevicePixelRatio();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)